config/global.ini.php

```diff
@@ -45,6 +45,9 @@
 ; Matomo should work correctly without this setting but we recommend to have a charset set.
 charset = utf8
 
+; disable dynamic row by default
```

Comment:
could we maybe explain a bit more why someone would want dynamic rows? eg mention it avoids "row size too large errors" for new tables. or maybe we could also mention that "we recommend enabling this setting if the default charset is set to utf8mb4".

cc @sgiehl  does that make sense?